conseil-api/src/main/resources/metadata/ethereum.mainnet.conf

```diff
@@ -0,0 +1,147 @@
+{
+  visible: false
+  entities {
+    blocks {
+      visible: true,
+      attributes {
+        hash {
+          visible: true
+        }
+        number {
+          visible: true
+        }
+        difficulty {
+          visible: true
+        }
+        extra_data {
+          visible: true
+        }
+        gas_limit {
+          visible: true
+        }
+        gas_used {
+          visible: true
+        }
+        logs_bloom {
+          visible: true
+        }
+        miner {
+          visible: true
+        }
+        mix_hash {
+          visible: true
+        }
+        nonce {
+          visible: true
+        }
+        parent_hash {
+          visible: true
+        }
+        receipts_root {
+          visible: true
+        }
+        sha3_uncles {
+          visible: true
+        }
+        size {
+          visible: true
+        }
+        state_root {
+          visible: true
+        }
+        total_difficulty {
+          visible: true
+        }
+        transactions_root {
+          visible: true
+        }
+        uncles {
+          visible: true
+        }
+        timestamp {
+          visible: true
+        }
+      }
+    }
+    transactions {
+      visible: true,
+      attributes {
+        hash {
+          visible: true
+        }
+        block_hash {
+          visible: true
+        }
+        block_number {
+          visible: true
+        }
+        from {
+          visible: true
+        }
+        gas {
+          visible: true
+        }
+        gas_price {
+          visible: true
+        }
+        input {
+          visible: true
+        }
+        nonce {
+          visible: true
+        }
+        to {
+          visible: true
+        }
+        transaction_index {
+          visible: true
+        }
+        value {
+          visible: true
+        }
+        v {
```

Comment:
just a question, does those letters mean anything specific? Maybe it would be a good idea to override their names with `name:`?

Reply:
Let me check documentation. Sounds like a good idea. Probably Scala-doc could be also extended to contain description for such fields.

Reply:
I managed to find the description (at least from the official wiki). Not sure if it's going to help but still... No entity for it, so no Scala-doc as well.

Reply:
sorry, my mistake, what I meant was `display-name` not `name`.

Reply:
That is correct. I should also verify that. Changed, thanks for letting know!